test/test_util.h

```diff
@@ -728,6 +728,43 @@ std::ostream& operator<<(std::ostream& os, const CUB_NS_QUALIFIER::KeyValuePair<
     return os;
 }
 
+#if CUB_IS_INT128_ENABLED
+static std::ostream& operator<<(std::ostream& os, __uint128_t val)
+{
+  constexpr int max_digits = 40;
+  char buffer[max_digits] = {};
+  char* digit = buffer + max_digits - 1;
+  const char* ascii = "0123456789";
+
+  do 
+  {
+    *digit = ascii[val % 10];
+    val /= 10;
+    digit--;
+  }
+  while(val != 0);
+
+  for (; digit != buffer + max_digits; digit++) {
+    os << *digit;
+  }
+
+  return os;
+}
+
+static std::ostream& operator<<(std::ostream& os, __int128_t val)
```

Comment:
Does this have to be defined manually? If yes, there should be tests for it.

Reply:
It's a test util only, therefore tests are welcome but not necessery.